### ISSUE TITLE
SafeInfinityUi's navigationBarBackground stopped working in 0.2.5

### ISSUE THREAD

**Issue by emvaized** [o4x/infinity_ui#4]
After update to 0.2.5 navigationBarBackground doesn't work anymore.

Widget tree like this:
```
return SafeInfinityUi(
        navigationBarBackground: Container(
                color: Colors.white,
              ),
            ...
```
Container is not displayed at all...

Android version is 9.

**Comment by emvaized** [comment]
Also `InfinityUi.navigationBarHeight` seems to be not always calculated correctly for some reason...

Reverting to 0.2.2 solves both of these issues. So seems like patches for Android 10's navbar, made in 0.2.5, broke something.

**Comment by o4x** [comment]
Hi @emvaized , I tested this in many device with different android versions But I did not encounter such a problem. Thank's for report this issue. please share your device name and if you can part of your code with me.

**Comment by emvaized** [comment]
@o4x Thanks for your reply!
Unfortunately, I can't test on a real device for now, so can confirm this happen only on emulator running Android 9.

**Comment by emvaized** [comment]
*closed issue accidentally*

**Comment by o4x** [comment]
Ohhh, I really don't know where the this problem comes from. What is your emulator ?

**Comment by emvaized** [comment]
@o4x Okay... Maybe I'm the only one facing this issue.

Here's emulator main specs:
![Screenshot_20200810_062102](https://user-images.githubusercontent.com/37851576/89749781-de405700-dad1-11ea-93c6-f28b7368c6de.png)

**Comment by o4x** [comment]
@emvaized Have you encountered this problem with the example that comes with this repository ? I tested with the this version of the emulator and did not encounter any problem. However, this issue will remain open.

**Comment by emvaized** [comment]
@o4x Yeah, I use basically the same layout, but with `Container` instead of `Image`...
Okay, thanks for your response anyway! Very cool and rare when such a great libraries are maintained that well. Keep it up :+1: 

I guess I'll stuck with 0.2.2 by now, since everything works more stable for me (except Android 10 navbar of course).
